src/components/RequestCard/index.tsx

```diff
@@ -19,6 +19,7 @@ import StatusBadge from '../StatusBadge';
 const messages = defineMessages({
   requestedby: 'Requested by {username}',
   seasons: 'Seasons',
+  all: 'ALL',
```

Comment:
Instead of capitalizing this here, lets instead make it `All` and then apply the `uppercase` text transform to the place we use it.

Reply:
Updated to All. Added uppercase to span